lib/tasks/ovirt.rake

```diff
@@ -0,0 +1,66 @@
+namespace :ovirt do
+  desc <<~END_DESC
+    Remove all Ovirt / RHEV compute resources & profiles from Foreman.
+    All hosts associated with these compute resources will be disassociated and kept.
+    All host groups with Ovirt compute resources & profiles will have their compute resource and compute profile removed.
+
+    Examples:
+      # foreman-rake ovirt:drop
+  END_DESC
+
+  task :drop => :environment do
+    compute_resources = ComputeResource.unscoped.where(type: 'Foreman::Model::Ovirt')
+
+    nothing_to_do if compute_resources.empty?
+
+    ask_user
+
+    puts 'Starting Ovirt integration drop ...'
+    compute_resources.each do |cr|
+      hosts = cr.hosts.unscoped
+      host_groups = cr.hostgroups.unscoped
+
+      hosts.each do |host|
+        puts "Disassociating host '#{host.name}' from the compute resource"
```

Comment:
You didn't log which compute resource. Perhaps add some `puts` at the start of the loop?